src/main/java/github/weichware10/toolbox/gui/Startbildschirm.java

```diff
@@ -0,0 +1,280 @@
+
+                // Request focus on the username field by default.
+                Platform.runLater(() -> url.requestFocus());
+
+                // nur bei Erfolg Dialog verlassen
+                final Button ok = (Button) dialog.getDialogPane().lookupButton(okButtonType);
+                ok.addEventFilter(ActionEvent.ACTION, okEvent -> {
+                    try {
+                        // Versuchen Datenbankclient zu erstellen
+                        dataBaseClient = new DataBaseClient(
+                                url.getText(),
+                                username.getText(),
+                                password.getText(),
+                                schema.getText());
+                        warningMessage.setVisible(false);
+                        startTestButton.setDisable(false);
+                    } catch (IllegalArgumentException e) {
+                        // display error
+                        Logger.info("Error while changing database connection");
+                        warning.setText("Your input is not valid:");
+                        // text benutzt um wrappingWidthProperty zu benutzen
+                        Text errorContent = new Text(e.getMessage());
+                        errorContent.wrappingWidthProperty().set(gridPane.getWidth() / 2);
+                        errorMessage.setContent(errorContent);
+                        errorMessage.setVisible(true);
+                        dialog.getDialogPane().setMinHeight(gridPane.getHeight());
+                        okEvent.consume();
+                    }
+                    // erstellt den Config Client um die Informationen aus der Config zu handeln
+                    configClient = new ConfigClient(dataBaseClient);
+                });
+
+                dialog.show();
+            }
+        });
+
+        // reset it
+        dataBaseReset.setOnAction(new EventHandler<ActionEvent>() {
+            @Override
+            public void handle(ActionEvent event) {
+                boolean databaseConnection = resetDataBaseConnection();
+                if (!databaseConnection) {
+                    warningMessage.setText(dataBaseWarning);
+                    warningMessage.setVisible(true);
+                    startTestButton.setDisable(true);
+                } else {
+                    warningMessage.setVisible(false);
+                    startTestButton.setDisable(false);
+                }
+            }
+        });
+
+        // Main Menüleiste
+        MenuBar testMenuBar = new MenuBar();
+        testMenuBar.getMenus().add(adminMenu);
+
+        // Layout welches alle Teillayouts zusammenfasst
+        BorderPane layoutTestIdWindow = new BorderPane();
+        layoutTestIdWindow.setTop(testMenuBar);
+        layoutTestIdWindow.setCenter(layoutTestIdCenterWindow);
+
+        toolBoxHome = new Scene(layoutTestIdWindow, 400, 400);
+
+        primaryStage.setScene(toolBoxHome);
+    }
+
+    /**
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.SummaryJavadocCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>First sentence of Javadoc is missing an ending period.